autode/bracket/ieip.py

```diff
@@ -0,0 +1,626 @@
+        method (Method): The method (low_sp keywords will be used)
+        n_cores (int): Total number of cores for all calculations
+
+    Returns:
+        (list[PotentialEnergy]): List of energies in order
+    """
+    n_cores_per_pp = max(n_cores // len(points), 1)
+    n_procs = min(n_cores, len(points))
+
+    with ProcessPool(max_workers=n_procs) as pool:
+        jobs = [
+            pool.submit(
+                _calculate_low_sp_energy_for_species,
+                species=point,
+                method=method,
+                n_cores=n_cores_per_pp,
+            )
+            for point in points
+        ]
+
+        energies = [job.result() for job in jobs]
+
+    return energies
+
+
+class ElasticImagePair(EuclideanImagePair):
+    """
+    This image-pair used for the Elastic Image Pair calculation. The
+    geometries after every macro-iteration are stored
+    """
+
+    @property
+    def last_left_step_size(self) -> float:
+        """The last step size on the left image"""
+        return np.linalg.norm(self.left_coords - self._left_history[-2])
+
+    @property
+    def last_right_step_size(self):
+        """The last step size on the right image"""
+        return np.linalg.norm(self.right_coords - self._right_history[-2])
+
+    @property
+    def ts_guess(self) -> Optional["Species"]:
+        """
+        Obtain the TS guess from the i-EIP image pair. The midpoint
+        between the two converged images is considered the TS guess
+
+        Returns:
+            (Species|None): The ts guess species, if images are available
+        """
+        # account for initial redistribution
+        if self.total_iters <= 2:
+            return None
+
+        tmp_spc = self._left_image.new_species(name="peak")
+        midpt_coords = np.array(self.left_coords + self.right_coords) / 2
+        tmp_spc.coordinates = midpt_coords
+        return tmp_spc
+
+    @property
+    def perpendicular_gradients(self):
+        perp_gradients = []
+        d_hat = self.dist_vec / np.linalg.norm(self.dist_vec)
+        for coord in [self.left_coords, self.right_coords]:
+            parall_g = d_hat * np.dot(d_hat, coord.g)
+            perp_g = coord.g - parall_g
+            rms_perp_g = np.sqrt(np.mean(np.square(perp_g)))
+            perp_gradients.append(rms_perp_g)
+
+        return perp_gradients
+
+    def redistribute_imagepair(
+        self,
+        ll_neb_interp: bool = True,
+        interp_fraction: float = 1 / 4,
+    ):
+        """
+        Redistribute the image pair by running a NEB calculation at lmethod or
+        use only IDPP interpolation, and then fitting a cubic spline on energy
+        calculated by the method (low_sp keywords). It generates the image pair
+        on both sides of the peak on the fitted spline, with a distance of
+        interp_fraction * total path distance on either side.
+
+        Args:
+            ll_neb_interp (bool): Whether to optimise the interpolated path with
```

Comment:
have you tested with this as `False`? a FF path might not be good enough to do sensible things

Reply:
@t-young31 Originally I was using just the IDPP path, which works well in most cases. For organic reactions I think low-level NEB optimisation will accelerate convergence but I am not entirely sure. The paper only uses IDPP.

Reply:
oh cool, so it's very NEB like